src/cascadia/TerminalApp/TerminalPage.xaml

```diff
@@ -58,6 +58,28 @@ the MIT License. See LICENSE in the project root for license information. -->
             x:Name="MultiLinePasteDialog"
             x:Uid="MultiLinePasteDialog"
             DefaultButton="Primary">
+            <StackPanel>
+                <TextBlock
+                    x:Name="MultiLineWarningText"
+                    x:Uid="MultiLineWarningText"
+                    TextWrapping="Wrap">
+                </TextBlock>
+                <TextBlock
+                    x:Name="ClipboardTextHeader"
+                    x:Uid="ClipboardTextHeader"
+                    Margin="0,16,0,0">
+                </TextBlock>
+                <ScrollViewer
+                    Margin="0,8,0,0"
+                    x:Name="ClipboardContentScrollViewer"
```

Comment:
We don't need to name infrastructural components unless we refer to them in code :smile:

Reply:
We actually refer to this one! To reset the scroll offset (the scroll offset does not reset automatically in between calls to open the dialog)